app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -25,8 +25,10 @@ class HomeFragment private constructor() : WolmoFragment<HomePresenter>(), HomeV
             )
         }
         vHomeTabs.setupWithViewPager(vHomeViewPager)
-        vHomeTabs.getTabAt(NEWS_TAB_INDEX)!!.setIcon(R.drawable.news_tab_icon)
-        vHomeTabs.getTabAt(PROFILE_TAB_INDEX)!!.setIcon(R.drawable.profile_tab_icon)
+        with(vHomeTabs) {
```

Comment:
you cna add thie line up of this one `vHomeTabs.setupWithViewPager(vHomeViewPager)` inside the with